src/methods.ts

```diff
@@ -127,7 +128,7 @@ async function upload_one_file(
       path.join('/upload/', name, artifactsPath),
       inputs.url
     ).toString()
-    return fileUpload(uploadUrl, inputs.user, inputs.password, file)
+    fileUpload(uploadUrl, inputs.user, inputs.password, file)
```

Comment:
Why are we stopping the return of the result? Even though I don't think we used it that much I'm curious to see if there's a reason  behind